spec/features/users/user_deletes_account_spec.rb

```diff
@@ -0,0 +1,36 @@
+# As an authenticated user
+# I want to delete my account
+# So that my information is no longer retained by the app
+
+
+feature 'user deletes account', %Q{
+  As a user, I want to delete my account
+  so that my information is no longer retained by the app
+} do
+
+  scenario 'user deletes account' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.